### PR TITLE
All commits

### PR DESCRIPTION
**Pull request by lukebaker**

### CODE REVIEW

File: replication_report-extensions.Rmd [hunk 1]
```diff
@@ -0,0 +1,320 @@
+---
+title: "Replication Report"
+author: "Yari & Kyonne" 
+date: "April 26, 2015"
+output: html_document
+---
+
```
Comment (msalganik) [inline replication_report-extensions.Rmd:7]: please add a bit of description at the beginning so that we can understand about your the context and goals of the replication

File: replication_report-extensions.Rmd [hunk 2]
```diff
@@ -0,0 +1,320 @@
...
+#View(dat)
+
+# Designate factors:
+dat$Groep <- as.factor(dat$Groep)
+dat$music <- as.factor(dat$music)
+dat$chose_advertised_pen <- as.factor(dat$chose_advertised_pen)
+
+# Filter out 2 excluded participants who accidentily took study twice in the paper:
+dat_e <- dat %>%
+  filter(excluded == 0)
```
Comment (msalganik) [inline replication_report-extensions.Rmd:53]: here do you mean to filter out people where excluded == 1?

File: replication_report-extensions.Rmd [hunk 3]
```diff
@@ -0,0 +1,320 @@
+---
+title: "Replication Report"
+author: "Yari & Kyonne" 
+date: "April 26, 2015"
+output: html_document
+---
+
+(1) Replicated table:
+
+```{r echo = FALSE}
```
Comment (msalganik) [inline replication_report-extensions.Rmd:10]: why echo = FALSE?  How can we see the output?

File: replication_report-extensions.Rmd [hunk 4]
```diff
@@ -0,0 +1,320 @@
...
+
+```
+
+(2) Explanation: 
+The table above is a reproduction of the frequencies and percentages of subjects' choice for advertised and non-advertised pen in the Liked Music & Disliked Music conditions, respectively, in Study 3. Note that at the bottom of the table is also the chi-square statistic, p-value and effect size (phi) for the data, indicating a medium-sized effect of music condition on pen choice. The table also demonstrates that the frequency of choice in the Disliked Music condition was in the hypothesized direction: Subjects tended to choose the non-advertised pen when made to listen to disliked music in an advertisement. Please note that the data represented above follows the exclusion criteria set forth in the published article. Specifically, any subjects who had participated in a prior version of the study were excluded (two subjects). In addition, 19 subjects were excluded for their so-called "deviant music evaluations". These subjects were foundd to either somewhat dislike the "liked music" (music attiude score below 3) or somewhat like the "disliked music" (music attitude score above 3).
+
+(3) Code for analyses & table:
```
Comment (msalganik) [inline replication_report-extensions.Rmd:36]: in general, below, it is hard for me to see what is attempting to replicate specific pieces from the paper and what is your own analysis

File: replication_report-extensions.Rmd [hunk 5]
```diff
@@ -0,0 +1,320 @@
...
+#CI95_up <- qchisq(.95, df = 1)
+#lower (5%)
+#CI95_low <- qchisq(.05, df = 1)
+
+## Replicate Table: Frequency table & chi-square statistic of subsetted data
+# (replicate "Experiment 3" portion of the table from article)
+
+# Generate count and proportion (percent) tallies for replicating the table:
+# Amongst those in Liked Music condition:
+dat_egoodLike <- dat_egood %>%
```
Comment (msalganik) [inline replication_report-extensions.Rmd:132]: is dat_egoodLike some kind of Dutch word?

File: replication_report-extensions.Rmd [hunk 6]
```diff
@@ -0,0 +1,320 @@
...
+# Generate table:
+Liked_Music <- c("21(57%)","16(43%)", "N=72, X^2(1)=8.59, p=0.003, phi=0.35")
+Disliked_Music <- c("8(23%)","27(77%)", "")
+Condition <- c("AdvertisedPen", "NonAdvertisedPen", "Chi-Square")
+Freq_Table <- cbind(Condition, Liked_Music, Disliked_Music)
+stargazer(Freq_Table, type = "text", title = "Study 3: Frequencies of Pen Choice for Each Conditon & Chi-Square")
+
+```
+
+(4.1) Extension A: Graphing the frequency of response & Investigating moderation by gender:
```
Comment (msalganik) [inline replication_report-extensions.Rmd:156]: I could not see this graphs.  For final report, please make sure they show up in html

File: replication_report-extensions.Rmd [hunk 7]
```diff
@@ -0,0 +1,320 @@
...
+                   labels=c("Liked", "Disliked")) +
+  theme_bw() + 
+  theme(axis.title.x = element_text(face = 'bold', size = 15),
+        axis.title.y = element_text(face = 'bold', size = 15), 
+        plot.title = element_text(face = 'bold', size = 20),
+        legend.title = element_text(face = 'bold', size = 12),
+        legend.text = element_text(size = 12),
+        axis.text.x = element_text(size = 12), 
+        axis.text.y = element_text(size = 12))
+```
```
Comment (msalganik) [inline replication_report-extensions.Rmd:179]: please clean-up indentation and spacing here

File: replication_report-extensions.Rmd [hunk 8]
```diff
@@ -0,0 +1,320 @@
...
+  theme_bw() + 
+  theme(axis.title.x = element_text(face = 'bold', size = 15),
+        axis.title.y = element_text(face = 'bold', size = 15), 
+        plot.title = element_text(face = 'bold', size = 20),
+        legend.title = element_text(face = 'bold', size = 12),
+        legend.text = element_text(size = 12),
+        axis.text.x = element_text(size = 12), 
+        axis.text.y = element_text(size = 12))
+
+```
```
Comment (msalganik) [inline replication_report-extensions.Rmd:206]: see above

File: replication_report-extensions.Rmd [hunk 9]
```diff
@@ -0,0 +1,320 @@
...
+
+Since the musical stimulus chosen for manipulation in this study was performed by a female artist, it is possible that the strength of aversion to the advertised pen (when paired with a disliked music) would be different by gender. Note: We are quite underpowered to conduct this test as a confirmatory analysis, and there is little theoretical precedent for looking at gender differences in pen choice; thus, this was an entirely exploratory investigation. It is possible that the disliked (and distorted sound) of the female singer's voice impacted women more so than men, such that women had an even greater liklihood of choosing the non-advertised pen when assigned to the disliked-music conditioin. However, as displayed in our analyses of the predicted probabilities below, first for the entire sample, and then for the data subsetted by gender, the likelihood of men and women chosing the advertised or non-advertised pen in each condition is nearly identitical. A greater sampling of men (as well as women) would be needed in order to test this hypothesis with greater statistical power.
+
```
Comment (msalganik) [inline replication_report-extensions.Rmd:211]: good extension and good summary

File: replication_report-extensions.Rmd [hunk 10]
```diff
@@ -0,0 +1,320 @@
...
+(4.2.1) A quick check of N by gender in data:
+
+```{r echo = FALSE}
+
+#Get counts and check values against article for correct coding scheme:
+count(dat_egood, geslacht1)
```
Comment (msalganik) [inline replication_report-extensions.Rmd:217]: if you don't want Dutch variable names, you can rename, and you can recode to female = 1 or 0.  That way you don't need to remember if men is 1 or 2

File: replication_report-extensions.Rmd [hunk 11]
```diff
@@ -0,0 +1,320 @@
...
+## 2 = female
+
+```
+
+(4.2.2) Code for logistical regressions:
+
+```{r eval = FALSE}
+
+#Recode dummy variable for music as 0 and 1 for easier interpretation of estimates:
+Music <- factor(dat_egood$music,
```
Comment (msalganik) [inline replication_report-extensions.Rmd:229]: do you use Music at all?

File: replication_report-extensions.Rmd [hunk 12]
```diff
@@ -0,0 +1,320 @@
...
+                    levels = c(1,2),
+                    labels = c("LikedMusic", "DislikedMusic"))
+
+dat_egood$music <- as.numeric(dat_egood$music)
+dat_egood <- dat_egood %>%
+  mutate(music.recode2 = (music - 1))
+View(dat_egood)
```
Comment (msalganik) [inline replication_report-extensions.Rmd:236]: don't leave this view in or it won't knit properly

File: replication_report-extensions.Rmd [hunk 13]
```diff
@@ -0,0 +1,320 @@
...
+
+#Set factor names:
+Music <- factor(dat_egood$music.recode2,
```
Comment (msalganik) [inline replication_report-extensions.Rmd:239]: why not name this disliked_music? Music could mean many different things.

File: replication_report-extensions.Rmd [hunk 14]
```diff
@@ -0,0 +1,320 @@
...
+
+ChosenPen <- factor(dat_egood$chose_advertised_pen,
+                levels = c(1, 0),
+                labels = c("ChoseAdPen", "NotChoseAdPen"))
+
+#Compare counts of each gender in paper to data for coding:
+count(dat_egood, geslacht1)
+## 1 = male
+## 2 = female
+
```
Comment (msalganik) [inline replication_report-extensions.Rmd:251]: why is this in the code twice?

File: replication_report-extensions.Rmd [hunk 15]
```diff
@@ -0,0 +1,320 @@
...
+# Males - Liked Music = 0.43
+exp(fitmale$estimate[1]) / (1 + exp(fitmale$estimate[1])) 
+
+# Females - Disliked Music = 0.77
+(exp(fitfemale$estimate[1] + fitfemale$estimate[2])) / 
+  (1 + exp(fitfemale$estimate[1] + fitfemale$estimate[2])) 
+# Males - Liked Music = 0.43
+exp(fitfemale$estimate[1]) / (1 + exp(fitfemale$estimate[1])) 
+
+
```
Comment (msalganik) [inline replication_report-extensions.Rmd:317]: can you pull all this together into a graph?

File: replication_report-extensions.Rmd (hunk 3 above)
Comment (ard1209) [inline replication_report-extensions.Rmd:10]: It would also be nice to see the original table next to your table. (See Kalyani and Megan's replication for an example).

File: replication_report-extensions.Rmd [hunk 16]
```diff
@@ -0,0 +1,320 @@
...
+# Load data
+dat <- read.csv("GORN_study3.csv", header = TRUE)
+dat <- tbl_df(dat)
+#head(dat)
+#View(dat)
+
+# Designate factors:
+dat$Groep <- as.factor(dat$Groep)
+dat$music <- as.factor(dat$music)
+dat$chose_advertised_pen <- as.factor(dat$chose_advertised_pen)
```
Comment (ard1209) [inline replication_report-extensions.Rmd:49]: Although this is a clear, a shorter variable name might be helpful

File: replication_report-extensions.Rmd [hunk 17]
```diff
@@ -0,0 +1,320 @@
...
+# Filter out 2 excluded participants who accidentily took study twice in the paper:
+dat_e <- dat %>%
+  filter(excluded == 0)
+#dat_e
+#View(dat_e)
+
+## Analyses on ENTIRE sample:
+
+# Test whether music condition is independent of pen choice:
+# (1 == liked music, 2 == disliked music)
```
Comment (ard1209) [inline replication_report-extensions.Rmd:60]: Why not make this is a dummy with 0 and 1 values like is normally seen and then rename the dummy in a meaningful way such as "liked_music"

File: replication_report-extensions.Rmd [hunk 18]
```diff
@@ -0,0 +1,320 @@
...
+# (0 == not chosen, 1 == chosen)
+tbl <- table(dat_e$music, dat_e$chose_advertised_pen) 
+tbl
```
Comment (ard1209) [inline replication_report-extensions.Rmd:63]: As Matt mentioned above, it is hard to follow what you are doing without begin able to see the output

File: WorkonModeration.R [hunk 19]
```diff
@@ -0,0 +1,219 @@
...
+## No correction but a Monte Carlo simulation:
+# chisq.test(tbl, simulate.p.value = TRUE)
+
+# Analyses on SUBSETTED sample:
+
+# Filter out additional 19 Ss with too positive attitudes in "disliked music" condition & too negative attitudes in "liked music" condition:
+
+# Find Cronbach's Alpha for music attitudes scale (for comparison with article):
+# dat_music <- dat_e %>%
+# select(att_muz1, att_muz2, att_muz3)
```
Comment (ard1209) [inline WorkonModeration.R:59]: The variable names here could be more meaningful

File: WorkonModeration.R [hunk 20]
```diff
@@ -0,0 +1,219 @@
...
+
+# Find Cronbach's Alpha for music attitudes scale (for comparison with article):
+# dat_music <- dat_e %>%
+# select(att_muz1, att_muz2, att_muz3) 
+# View(dat_music)
+# alpha(dat_music, na.rm = TRUE, delete=TRUE)
+
+#Find mean attitudes for each music condition (for comparison with article): 
+dat_e %>%
+  group_by(music) %>%
```
Comment (ard1209) [inline WorkonModeration.R:65]: Music is also an unclear name as I mentioned above

File: WorkonModeration.R [hunk 21]
```diff
@@ -0,0 +1,219 @@
...
+library(stargazer)
+
+
+#Create dummy variables:
+Music <- factor(dat_egood$music,
+                    levels = c(1,2),
+                    labels = c("LikedMusic", "DislikedMusic"))
+
+
+#Recode dummy variables:
```
Comment (ard1209) [inline WorkonModeration.R]: Good

File: WorkonModeration.R [hunk 22]
```diff
@@ -0,0 +1,219 @@
...
+dat_egood$music <- as.numeric(dat_egood$music)
+dat_egood <- dat_egood %>%
+  mutate(music.recode2 = (music - 1))
```
Comment (ard1209) [inline WorkonModeration.R:126]: Good step, but the naming is still unclear

File: WorkonModeration.R [hunk 23]
```diff
@@ -0,0 +1,219 @@
...
+View(dat_egood)
+
+#Set factors:
+Music <- factor(dat_egood$music.recode2,
```
Comment (ard1209) [inline WorkonModeration.R:130]: A better name would be "disliked_music"

File: WorkonModeration.R [hunk 24]
```diff
@@ -0,0 +1,219 @@
...
+                levels = c(1,0),
+                labels = c("ChoseAdPen", "NotChoseAdPen"))
+
+#Compare counts of each gender in paper to data for coding:
+count(dat_egood, geslacht1)
+## 1 = male
+## 2 = female
+
+# need to check which is which:
+Gender <- factor(dat_egood$geslacht1,
```
Comment (ard1209) [inline WorkonModeration.R:144]: I suggest you make this a 0, 1 dummy and name it whatever you assign 1 to be

File: WorkonModeration.R [hunk 25]
```diff
@@ -0,0 +1,219 @@
...
+                    labels = c("Male", "Female"))
+
+# for relationship between advertised pen and music, music 2 (disliked) seems significant:
+fit1 <- glm(ChosenPen ~ Music, data = dat_egood, family = binomial)
+fit1 <- tidy(fit1)
+fit1
+
+#### calculate odds 
+fit1$estimate
+fit1$estimate[1] + fit1$estimate[2] ## 1.216
```
Comment (ard1209) [inline WorkonModeration.R:155]: Try to call these my name, not position

File: WorkonModeration.R [hunk 26]
```diff
@@ -0,0 +1,219 @@
...
+(exp(fitmale$estimate[1] + fitmale$estimate[2])) / 
+  (1 + exp(fitmale$estimate[1] + fitmale$estimate[2])) 
+# Males - Liked Music = 0.43
+exp(fitmale$estimate[1]) / (1 + exp(fitmale$estimate[1])) 
+
+# Females - Disliked Music = 0.77
+(exp(fitfemale$estimate[1] + fitfemale$estimate[2])) / 
+  (1 + exp(fitfemale$estimate[1] + fitfemale$estimate[2])) 
+# Males - Liked Music = 0.43
+exp(fitfemale$estimate[1]) / (1 + exp(fitfemale$estimate[1]))
```
Comment (ard1209) [inline WorkonModeration.R:207]: see comment above for this entire section

File: WorkonModeration.R [hunk 27]
```diff
@@ -0,0 +1,219 @@
...
+# of choosing the advertised pen when listening to liked versus disliked music.
+# Males - Disliked Music = 0.77
+(exp(fitmale$estimate[1] + fitmale$estimate[2])) / 
+  (1 + exp(fitmale$estimate[1] + fitmale$estimate[2])) 
+# Males - Liked Music = 0.43
+exp(fitmale$estimate[1]) / (1 + exp(fitmale$estimate[1])) 
+
+# Females - Disliked Music = 0.77
+(exp(fitfemale$estimate[1] + fitfemale$estimate[2])) / 
+  (1 + exp(fitfemale$estimate[1] + fitfemale$estimate[2]))
```
Comment (ard1209) [inline WorkonModeration.R:205]: a function would be helpful here since you are repeating your code

File: WorkonModeration.R (hunk 26 above)
Comment (ard1209) [inline WorkonModeration.R:207]: Oops just realized I was commenting on the wrong file. Oh well :)

File: replication_report-extensions.Rmd [hunk 28]
```diff
@@ -0,0 +1,320 @@
...
+# Power analysis -- Find the effect size for a test with .8 sensitivity:
+pwr.chisq.test(w = NULL, N = 91, df = 1, sig.level = 0.05, power = .8)
+
+# Analyses on SUBSETTED sample:
+
+# Filter out additional 19 Ss with too positive attitudes in "disliked music" condition & too negative attitudes in "liked music" condition:
+
+# Find Cronbach's Alpha for music attitudes scale (for comparison with article):
+#dat_music <- dat_e %>%
+#  select(att_muz1, att_muz2, att_muz3)
```
Comment (ard1209) [inline replication_report-extensions.Rmd:80]: Naming could be clearer

File: replication_report-extensions.Rmd [hunk 29]
```diff
@@ -0,0 +1,320 @@
...
+#View(dat_music)
+#alpha(dat_music, na.rm = TRUE, delete=TRUE)
+
+# Find mean attitudes for each music condition (for comparison with article): 
+#dat_e %>%
+#  group_by(music) %>%
```
Comment (ard1209) [inline replication_report-extensions.Rmd:86]: Should be either something like liked_music or dislike_music. It's unclear what this is as it stands

File: replication_report-extensions.Rmd [hunk 30]
```diff
@@ -0,0 +1,320 @@
...
+```{r eval= FALSE}
+
+# Load data
+dat <- read.csv("GORN_study3.csv", header = TRUE)
+dat <- tbl_df(dat)
+#head(dat)
+#View(dat)
+
+# Designate factors:
+dat$Groep <- as.factor(dat$Groep)
```
Comment (ard1209) [inline replication_report-extensions.Rmd:47]: what does this mean?

File: replication_report-extensions.Rmd [hunk 31]
```diff
@@ -0,0 +1,320 @@
...
+dat_egoodLike <- dat_egood %>%
+  filter(music == 1)
+dat_egoodLike %>%
+  group_by(chose_advertised_pen) %>%
+  summarise(n = n(),
+            percent = n / nrow(dat_egoodLike) * 100)
+
+# Amongst those in Disliked Music condition:
+dat_egoodDislike <- dat_egood %>%
+  filter(music == 2)
```
Comment (ard1209) [inline replication_report-extensions.Rmd:141]: Since this is not a 0, 1 dummy it is hard to know what 2 means. Although the comment is helpful

File: replication_report-extensions.Rmd [hunk 32]
```diff
@@ -0,0 +1,320 @@
...
+                    labels = c("Male", "Female"))
+
+# for relationship between advertised pen and music, music 2 (disliked) seems significant:
+fit1 <- glm(ChosenPen ~ Music, data = dat_egood, family = binomial)
+fit1 <- tidy(fit1)
+fit1
+
+#### calculate odds 
+fit1$estimate
+fit1$estimate[1] + fit1$estimate[2] ## 1.216
```
Comment (ard1209) [inline replication_report-extensions.Rmd:264]: Can you call this by name, not position?

File: replication_report-extensions.Rmd [hunk 33]
```diff
@@ -0,0 +1,320 @@
...
+## odds ratio
+exp(fit1$estimate[2]) ##4.43
+## predicted probabilities:
+# Disliked Music:
+(exp(fit1$estimate[1] + fit1$estimate[2])) / (1 + exp(fit1$estimate[1] + fit1$estimate[2]))
+# Liked Music: 
+exp(fit1$estimate[1]) / (1 + exp(fit1$estimate[1]))
```
Comment (ard1209) [inline replication_report-extensions.Rmd:271]: See above comment

File: replication_report-extensions.Rmd [hunk 34]
```diff
@@ -0,0 +1,320 @@
...
+# of choosing the advertised pen when listening to liked versus disliked music.
+# Males - Disliked Music = 0.77
+(exp(fitmale$estimate[1] + fitmale$estimate[2])) / 
+  (1 + exp(fitmale$estimate[1] + fitmale$estimate[2])) 
+# Males - Liked Music = 0.43
+exp(fitmale$estimate[1]) / (1 + exp(fitmale$estimate[1])) 
+
+# Females - Disliked Music = 0.77
+(exp(fitfemale$estimate[1] + fitfemale$estimate[2])) / 
+  (1 + exp(fitfemale$estimate[1] + fitfemale$estimate[2]))
```
Comment (ard1209) [inline replication_report-extensions.Rmd:313]: Since you do this multiple times perhaps you should write a function for calculating the predicted probabiltiies

File: replication_report.Rmd [hunk 35]
```diff
@@ -0,0 +1,156 @@
+---
```
Comment (ard1209) [inline replication_report.Rmd:1]: What is this file?

File: replication_report-extensions.Rmd [hunk 36]
```diff
@@ -0,0 +1,320 @@
+---
+title: "Replication Report"
+author: "Yari & Kyonne" 
+date: "April 26, 2015"
+output: html_document
```
Comment (lolNickFox) [inline replication_report-extensions.Rmd:5]: is this file your replication and extension? the file name makes me think it's just for the extension, but I couldn't find a file for just the replication.  A clearer filename would help.

File: replication_report-extensions.Rmd (hunk 3 above)
Comment (lolNickFox) [inline replication_report-extensions.Rmd:10]: I understand echo = false, because it was probably hiding the code and producing the stargazer table, right?  I used it like that to hide the gears that were churning out figures/summaries that would still show when knitted up.

File: replication_report-extensions.Rmd [hunk 37]
```diff
@@ -0,0 +1,320 @@
...
+suppressPackageStartupMessages(require(tidyr))
+suppressPackageStartupMessages(require(pwr)) 
+suppressPackageStartupMessages(require(broom))
+suppressPackageStartupMessages(require(lsr))
+suppressPackageStartupMessages(require(ggplot2))
+suppressPackageStartupMessages(require(MBESS))
+suppressPackageStartupMessages(require(psych))
+suppressPackageStartupMessages(require(car))
+suppressPackageStartupMessages(require(MASS))
+suppressPackageStartupMessages(require(stargazer))
```
Comment (lolNickFox) [inline replication_report-extensions.Rmd:23]: It would be helpful to comment in what some of the more unusual packages were for, like MBESS and psych?

File: replication_report-extensions.Rmd [hunk 38]
```diff
@@ -0,0 +1,320 @@
...
+
+(3) Code for analyses & table:
+
+```{r eval= FALSE}
+
+# Load data
+dat <- read.csv("GORN_study3.csv", header = TRUE)
+dat <- tbl_df(dat)
+#head(dat)
+#View(dat)
```
Comment (lolNickFox) [inline replication_report-extensions.Rmd:44]: Here is another "view" that might mess up your knitting

File: replication_report-extensions.Rmd [hunk 39]
```diff
@@ -0,0 +1,320 @@
...
+# Designate factors:
+dat$Groep <- as.factor(dat$Groep)
+dat$music <- as.factor(dat$music)
+dat$chose_advertised_pen <- as.factor(dat$chose_advertised_pen)
+
+# Filter out 2 excluded participants who accidentily took study twice in the paper:
+dat_e <- dat %>%
+  filter(excluded == 0)
+#dat_e
+#View(dat_e)
```
Comment (lolNickFox) [inline replication_report-extensions.Rmd:55]: here's another "view" - just pointing them out if they're causing knitting problems

File: replication_report-extensions.Rmd (hunk 17 above)
Comment (lolNickFox) [inline replication_report-extensions.Rmd:60]: Our paper also had data that was dummy coded 1 and 2, not 0 and 1.  Is it understood that we should change this while replicating?

File: replication_report-extensions.Rmd [hunk 40]
```diff
@@ -0,0 +1,320 @@
...
+# Chi-Square, no correction, no simulation:
+cs1 <- chisq.test(tbl, correct = FALSE)
+
+# Power analysis -- Find the power to detect .35 effect size reported in paper:
+pwr.chisq.test(w = .39, N = 91, df = 1, sig.level = 0.05, power = NULL)
+
+# Power analysis -- Find the effect size for a test with .8 sensitivity:
+pwr.chisq.test(w = NULL, N = 91, df = 1, sig.level = 0.05, power = .8)
+
+# Analyses on SUBSETTED sample:
```
Comment (lolNickFox) [inline replication_report-extensions.Rmd]: how is this different than "ENTIRE" sample?

File: replication_report-extensions.Rmd [hunk 41]
```diff
@@ -0,0 +1,320 @@
...
+
+# Filter out additional 19 Ss with too positive attitudes in "disliked music" condition & too negative attitudes in "liked music" condition:
+
+# Find Cronbach's Alpha for music attitudes scale (for comparison with article):
+#dat_music <- dat_e %>%
+#  select(att_muz1, att_muz2, att_muz3)
+#View(dat_music)
```
Comment (lolNickFox) [inline replication_report-extensions.Rmd:81]: another "view"

File: replication_report-extensions.Rmd [hunk 42]
```diff
@@ -0,0 +1,320 @@
...
+pwr.chisq.test(w = .35, N = 72, df = 1, sig.level = 0.05, power = NULL)
+
+# Power analysis -- Find the effect size for a test with minimum of .8 sensitivity:
+pwr.chisq.test(w = NULL, N = 72, df = 1, sig.level = 0.05, power = .8)
+
+# Confidence Intervals (not replicated from paper):
+#upper (95%)
+#CI95_up <- qchisq(.95, df = 1)
+#lower (5%)
+#CI95_low <- qchisq(.05, df = 1)
```
Comment (lolNickFox) [inline replication_report-extensions.Rmd:125]: I found the best way to replicate 95% confidence intervals was just to multiply 1.96 \* the SEM., because if you think about what the 95% confidence interval is, it's 1.96 standard deviations of the sampling distribution (in both directions).  When we tried using a baked in function in R, it was always wrong, but 1.96 \* SEM always worked.

File: replication_report-extensions.Rmd (hunk 7 above)
Comment (lolNickFox) [inline replication_report-extensions.Rmd:179]: could also break up into multiple parts for clarity

File: replication_report-extensions.Rmd [hunk 43]
```diff
@@ -0,0 +1,320 @@
...
+        axis.title.y = element_text(face = 'bold', size = 15), 
+        plot.title = element_text(face = 'bold', size = 20),
+        legend.title = element_text(face = 'bold', size = 12),
+        legend.text = element_text(size = 12),
+        axis.text.x = element_text(size = 12), 
+        axis.text.y = element_text(size = 12))
+
+```
+
+(4.2) Extension B: Investigating the possible moderation of effects by gender, using a logistical regression.
```
Comment (lolNickFox) [inline replication_report-extensions.Rmd:208]: Great explanation for what is coming next!

File: replication_report-extensions.Rmd [hunk 44]
```diff
@@ -0,0 +1,320 @@
...
+## 1 = male
+## 2 = female
+
+```
+
+(4.2.2) Code for logistical regressions:
+
+```{r eval = FALSE}
+
+#Recode dummy variable for music as 0 and 1 for easier interpretation of estimates:
```
Comment (lolNickFox) [inline replication_report-extensions.Rmd:228]: this is good, try to use it earlier in the code too (where gender was coded 1 and 2)

File: replication_report-extensions.Rmd [hunk 45]
```diff
@@ -0,0 +1,320 @@
...
+# Females - Disliked Music = 0.77
+(exp(fitfemale$estimate[1] + fitfemale$estimate[2])) / 
+  (1 + exp(fitfemale$estimate[1] + fitfemale$estimate[2])) 
+# Males - Liked Music = 0.43
+exp(fitfemale$estimate[1]) / (1 + exp(fitfemale$estimate[1])) 
+
+
+
+```
+Note that the `echo = FALSE` parameter was added to the code chunk to prevent printing of the R code that generated the table. Further, the "eval = FALSE" parameter was added to the top code chunk to prevent printing of the code.
```
Comment (lolNickFox) [inline replication_report-extensions.Rmd:320]: as I suspected! - could put this at that chunk =)

File: replication_report-extensions.Rmd [hunk 46]
```diff
@@ -0,0 +1,320 @@
...
+
+# Power analysis -- Find the power to detect .35 effect size reported in paper:
+pwr.chisq.test(w = .39, N = 91, df = 1, sig.level = 0.05, power = NULL)
+
+# Power analysis -- Find the effect size for a test with .8 sensitivity:
+pwr.chisq.test(w = NULL, N = 91, df = 1, sig.level = 0.05, power = .8)
+
+# Analyses on SUBSETTED sample:
+
+# Filter out additional 19 Ss with too positive attitudes in "disliked music" condition & too negative attitudes in "liked music" condition:
```
Comment (lolNickFox) [inline replication_report-extensions.Rmd:76]: is this how the SUBSETTED data is different than the ENTIRE data? how is "too positive" and "too negative" operationalized?

File: WorkonModeration.R [hunk 47]
```diff
@@ -0,0 +1,219 @@
...
+#head(dat)
+#View(dat)
+
+# Designate factors:
+dat$Groep <- as.factor(dat$Groep)
+dat$music <- as.factor(dat$music)
+dat$chose_advertised_pen <- as.factor(dat$chose_advertised_pen)
+
+## Filter out 2 excluded participants who accidentily took study twice in the paper:
+dat_e <- dat %>%
```
Comment (kalmontjay) [inline WorkonModeration.R:25]: Possible to choose a more meaningful data name?

File: replication_report-extensions.Rmd (hunk 38 above)
Comment (kalmontjay) [inline replication_report-extensions.Rmd:44]: I wonder if there are other checks you could do other than view the data. I think it's great that it's commented out though, so it won't show up in the R-Markdown but it's still available for someone to see. I am curious why so many lines of code are commented out? Are those lines of codes specifically meant to be checks or is it because of knitting problems?

File: replication_report-extensions.Rmd (hunk 41 above)
Comment (kalmontjay) [inline replication_report-extensions.Rmd:81]: So many views!

File: replication_report-extensions.Rmd (hunk 36 above)
Comment (kalmontjay) [inline replication_report-extensions.Rmd:5]: Agreed, I think I might have commented on four different files.

File: replication_report-extensions.Rmd (hunk 4 above)
Comment (kalmontjay) [inline replication_report-extensions.Rmd:36]: Might be a good idea to directly copy and paste (with proper citations of course) descriptions of figures/tables from the paper.

File: replication_report-extensions.Rmd [hunk 48]
```diff
@@ -0,0 +1,320 @@
...
+
+# Find Cronbach's Alpha for music attitudes scale (for comparison with article):
+#dat_music <- dat_e %>%
+#  select(att_muz1, att_muz2, att_muz3)
+#View(dat_music)
+#alpha(dat_music, na.rm = TRUE, delete=TRUE)
+
+# Find mean attitudes for each music condition (for comparison with article):
```
Comment (kalmontjay) [inline replication_report-extensions.Rmd:84]: Is this just a check or are you using these for something? If it is the latter, might be better to save as a dataframe

File: replication_report-extensions.Rmd (hunk 45 above)
Comment (kalmontjay) [inline replication_report-extensions.Rmd:320]: The extension was a really great idea! I really feel like you took this analysis further in the sense that you tried to test an interesting and meaningful hypothesis. It would be better if your files had clearer descriptions and they was more description of specifically what you were trying to replicate. It would also have been useful to compare your replication to what the authors had done.

File: replication_report-extensions.Rmd [hunk 49]
```diff
@@ -0,0 +1,320 @@
...
+  filter(geslacht1 == "2")
+#count(female.dat_egood, geslacht1)
+
+# MALES:
+fitmale <- glm(ChosenPen ~ Music, data = male.dat_egood, family=binomial)
+fitmale <- tidy(fitmale)
+fitmale
+#### calculate odds 
+fitmale$estimate
+fitmale$estimate[1] + fitmale$estimate[2] ## 1.216
```
Comment (kalmontjay) [inline replication_report-extensions.Rmd:289]: It seems unnecessary to report the output in a comment. Couldn't you just return the output. In the spirit of transparency, I would trust what you were saying more if I could actually see the output.

File: replication_report-extensions.Rmd [hunk 50]
```diff
@@ -0,0 +1,320 @@
...
+# Amongst those in Disliked Music condition:
+dat_egoodDislike <- dat_egood %>%
+  filter(music == 2)
+dat_egoodDislike %>%
+  group_by(chose_advertised_pen) %>%
+  summarise(n = n(),
+            percent = n / nrow(dat_egoodDislike) * 100)
+
+# Generate table:
+Liked_Music <- c("21(57%)","16(43%)", "N=72, X^2(1)=8.59, p=0.003, phi=0.35")
```
Comment (kalmontjay) [inline replication_report-extensions.Rmd:148]: I don't think vector names are supposed to be capitalized according to the style guide.

File: replication_report-extensions.Rmd (hunk 37 above)
Comment (kalmontjay) [inline replication_report-extensions.Rmd:23]: Might seem silly and very trivial but I think there is a (heated) debate about using require vs. library. According to this post, library is a better option for loading packages: http://www.r-bloggers.com/library-vs-require-in-r/

File: replication_report-extensions.Rmd [hunk 51]
```diff
@@ -0,0 +1,320 @@
...
+cs2 <- chisq.test(tbl2, correct = FALSE)
+tidy(cs2)
+
+# Power analysis -- Find the power to detect .35 effect size reported in paper:
+pwr.chisq.test(w = .35, N = 72, df = 1, sig.level = 0.05, power = NULL)
+
+# Power analysis -- Find the effect size for a test with minimum of .8 sensitivity:
+pwr.chisq.test(w = NULL, N = 72, df = 1, sig.level = 0.05, power = .8)
+
+# Confidence Intervals (not replicated from paper):
```
Comment (kalmontjay) [inline replication_report-extensions.Rmd:121]: Unclear what this means. Does it mean that the paper doesn't use confidence intervals or that you couldn't replicate the results?

File: replication_report-extensions.Rmd [hunk 52]
```diff
@@ -0,0 +1,320 @@
...
+
+# Find mean attitudes for each music condition (for comparison with article): 
+#dat_e %>%
+#  group_by(music) %>%
+#  summarise(avg = mean(c(att_muz1, att_muz2, att_muz3)))
+
+# Add mean attitude column to dataframe:
+dat_e <- dat_e %>%
+  mutate(avg_att_muz = (att_muz1 + att_muz2 + att_muz3) / 3)
+#View(dat_e)
```
Comment (kalmontjay) [inline replication_report-extensions.Rmd:92]: Another view

File: replication_report-extensions.Rmd (hunk 28 above)
Comment (kalmontjay) [inline replication_report-extensions.Rmd:80]: Why is this commented out?

File: replication_report-extensions.Rmd (hunk 7 above)
Comment (kalmontjay) [inline replication_report-extensions.Rmd:179]: Might be a good idea to save these graphs both as images as well as with object names.
